db/config.cc

```diff
@@ -907,6 +907,12 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , task_ttl_seconds(this, "task_ttl_in_seconds", liveness::LiveUpdate, value_status::Used, 10, "Time for which information about finished task stays in memory.")
     , cache_index_pages(this, "cache_index_pages", liveness::LiveUpdate, value_status::Used, true,
         "Keep SSTable index pages in the global cache after a SSTable read. Expected to improve performance for workloads with big partitions, but may degrade performance for workloads with small partitions.")
+    , wasm_cache_size(this, "wasm_cache_size", value_status::Used, 128 * 1024 * 1024, "Maximum total size of all WASM instances stored in the cache")
```

Comment:
is it per shard? shouldn't we make it a percentage of total memory available? then it automatically adjust based on memory available for each shard.